WorkloadManagementSystem/Agent/SiteDirector.py

```diff
@@ -531,7 +531,7 @@ def submitJobs( self ):
       if not result['OK']:
         return result
       self.proxy = result['Value']
-      ce.setProxy( self.proxy, cpuTime - 60 )
+      ce.setProxy( self.proxy, self.proxy.getRemainingSecs()['Value'] )
```

Comment:
It is generally not a good practice to not check the return status of the method. If even it can never return an error, the fact that the developer returns an S_OK assumes that eventually the implementation can be more sophisticated and may return S_ERROR as well